.github/workflows/release.yml

```diff
@@ -7,6 +7,11 @@ on:
         description: 'Version'
         required: true
         type: string
+      debian_revision:
+        description: 'Debian revision'
+        required: true
+        default: '1'
+        type: string

```

Comment:
Is the Debian revision something we expect to need to change as a workflow parameter? Or would this make more sense as a hard coded env var for the whole workflow?

Reply:
Also, a comment about what this is would be nice `#` can be used for a comment line in YAML.

Reply:
The probability is pretty low.

Reply:
@kyle-rader-msft added a comment. Does this looks okay?

Reply:
AHA That's a very important comment and I think we could add more detail. It sounds like this is a revision number designed for the same reason we have `-rev-n` on the AzureAuth-Lib pacakge, so we can rev the package without reving the azureauth version it's associated with.

So, in theory - we would set the revision to 2, if we needed to release a new debian package, but the source code of AzureAuth itself had not changed, correct?

Reply:
yes that's exactly right.

Reply:
Added more details in the comment.